Makefile

```diff
@@ -306,23 +306,19 @@ verify: vendor verify-codegen verify-mockgen verify-manifests #HELP Run all veri
 
 #SECTION Release
 
-.PHONY: pull-opm
-pull-opm:
-	docker pull $(OPERATOR_REGISTRY_IMAGE)
-
 .PHONY: package
 package: $(YQ) $(HELM) #HELP Package OLM for release
-package: OLM_RELEASE_IMG_REF=$(shell docker inspect --format='{{index .RepoDigests 0}}' $(IMAGE_REPO):$(RELEASE_VERSION))
-package: OPM_IMAGE_REF=$(shell docker inspect --format='{{index .RepoDigests 0}}' $(OPERATOR_REGISTRY_IMAGE))
+package: OLM_RELEASE_IMG_REF=$(shell go run util/image-canonical-ref/main.go $(IMAGE_REPO):$(RELEASE_VERSION))
+package: OPM_IMAGE_REF=$(shell go run util/image-canonical-ref/main.go $(OPERATOR_REGISTRY_IMAGE))
```

Comment:
dumb question: would we bail on failure here?

Reply:
Good question! It used to fail on `docker pull`, but now that it is gone - we should probably add extra checks. Or return `docker pull`s. I'l look into it.

Reply:
I reverted most of the changes and switched to `docker images` instead of `docker inspect`. So this is no longer a concern: we will fail on `docker pull` should something o wrong.